src/MatBlazor/Components/MatAutocompleteList/BaseMatAutocompleteList.cs

```diff
@@ -190,7 +194,6 @@ public async void OnKeyDown(KeyboardEventArgs ev)
             if (ev.Key == "ArrowDown" || ev.Key == "ArrowUp")
             {
                 int currentIndex = await ListRef.GetSelectedIndex();
-                int nextIndex = (ev.Key == "ArrowDown") ? currentIndex++ : currentIndex--;
```

Comment:
Removing this line seems to break the up and down arrows in it?

Reply:
Maybe it should be

`if (ev.Key == "ArrowDown") currentIndex++; else currentIndex--;`

Then we don't have an unused "nextIndex" variable?

Reply:
Thanks, I changed it to 
                ```
var currentIndex = await ListRef.GetSelectedIndex();
                await ListRef.SetSelectedIndex(ev.Key == "ArrowDown" ? currentIndex + 1 : currentIndex -1);
```
to be more readable and not have an unused nextIndex